viahtml/context.py

```diff
@@ -91,6 +91,13 @@ def proxied_url_with_config(self):
 
         return url if url else None
 
+    @property
+    @lru_cache(1)
+    def via_config(self):
```

Comment:
Oh, is this the first time we use `via.` params over viahtml? Interesting

Reply:
`via.*` params are read indirectly in template variables, see `Hooks.template_variables`. That function does not have access to the `Context` object however because it is called on app startup, when the Jinja environment is configured. It does however have access to an `env` object which I believe is the same as `Context.http_environ`.